pkg/sources/reconciler/common/labels.go

```diff
@@ -37,3 +37,8 @@ const (
 	managedBy        = "triggermesh-controller"
 	componentAdapter = "adapter"
 )
+
+// LabelsPropagationList is the list of labels that should be propagated to the adapters.
```

Comment:
The linter didn't complain because this var is private, but this comment should start with a lowercase letter.